src/grt/src/fastroute/src/maze.cpp

```diff
@@ -1044,7 +1044,6 @@ void reInitTree(int netID)
     if (treeedge->len > 0) {
       delete[] treeedge->route.gridsX;
       delete[] treeedge->route.gridsY;
-      delete[] treeedge->route.gridsL;
```

Comment:
Why is this wrong?

Reply:
gridsL is not initialized until 3D routing, so I think it was deleting a nullptr.

Reply:
Ok though deleting a nullptr is not an error.  More likely it was uninitialized.